src/components/Button/Button.tsx

```diff
@@ -4,13 +4,14 @@ import styles from './Button.module.css';
 import { Icon } from '../Icon/Icon';
 export interface Props {
   /**
-   * Button reference
+   * Visually hidden button text (but text is still accessible to assistive technology).
+   * This overrides `text`
    */
-  buttonRef?: any;
+  'aria-label'?: string;
```

Comment:
I added `aria-label` to the props just in case since I'm removing the other hidden text props.